janusgraph-core/src/main/java/org/janusgraph/diskstorage/configuration/TransactionalConfiguration.java

```diff
@@ -68,7 +68,7 @@ public WriteConfiguration copy() {
     @Override
     public Iterable<String> getKeys(final String prefix) {
         return Iterables.concat(
-        Iterables.filter(writtenValues.keySet(), s -> s.startsWith(prefix)),
+        Iterables.filter(writtenValues.keySet(), s -> s != null && s.startsWith(prefix)),
```

Comment:
(nitpick) Is `null` key here is possible situation? I understand that HaskMap allows `null` key but I don't know if the logic will handle this situation.